azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -264,7 +269,6 @@ def update_device_with_sas(
         kwargs = {
             "device_id": device_id,
             "status": status,
-            "etag": etag,
```

Comment:
etags have been removed from any request bodies. From my testing, putting etags in the body have no effect and they should only be in headers. I assume the etags were added to the request body due to confusing documentation [like this](https://docs.microsoft.com/en-us/rest/api/iothub/service/devices/createorupdateidentity) which shows etags as part of the request body. It seems like the reason why that property shows up in the documentation is because it was in a schema in the swagger.